app/models/specialist_document.rb

```diff
@@ -86,7 +86,11 @@ def update(params)
   def published?
     editions.any?(&:published?)
   end
-  alias_method :has_ever_been_published?, :published?
+
+  def has_ever_been_published?
+    return false if @editions.size == 1 && needs_exporting?
+    published?
```

Comment:
And, this particular problem is really only a problem for the first edition of a document.  In the past the UI would let you create a minor update for the first edition but we have to tell the publishing-api that it's a major update the first time it's published - regardless of what the model thinks.  The UI no longer allows this, but historical data still persists and it can break republish events if we don't cope with it.